src/script/ceph-release-notes

```diff
@@ -62,6 +63,19 @@ def make_release_notes(gh, repo, ref, plaintext):
                 print ("ERROR: http://github.com/ceph/ceph/pull/" + str(number) + " has no associated issue")
                 continue    
 
+            if original_issue:
+                r = requests.get(tracker_uri.format(issue),params={"include":"relations"})
+                # Now we can get an original issue only if the
+                # relations field is populated, otherwise we fallback
+                # to the tracker number
+                try:
+                    relations = r.json()["issue"]["relations"]
+                except KeyError:
+                    break
+                related_issues = [str(i['issue_id']) for i in relations if i["relation_type"] == "copied_to"]
```

Comment:
I'm hoping the relations key would error out then... or the filter ie `copied_to` would fail... if both of this fail then I guess we can add a check for "the issue tracker is backports" or something. WDYT?